### ISSUE TITLE
Mobile Support

### ISSUE THREAD

**Issue by ayunami2000** [lionleaf/dwitter#303]
Please check mobile support, especially on iOS devices with a screen size the same as an iPhone SE. It doesn't quite work right. I can't get the screen recording here right now, so try it yourself

**Comment by dwitterer** [comment]
I have.. You're gonna need to give us more detail if you want your issue resolved and closed.

**Comment by dwitterer** [comment]
Actually, scrap that...

![screenshot from 2018-03-26 20 46 24](https://user-images.githubusercontent.com/36137355/37929071-cfa9739e-3136-11e8-96ad-fc4b1e78c6da.png)

Screencap for lazy people...

Awesome button and new dweet button are cropped or stretched... Menu overflow overlaps content and needs to be made more adaptive on *all* platforms... It is known. But yeah, at least we have an iisue for it now :D

**Comment by stianjensen** [comment]
I'll push some fixes to make this a bit better!

**Comment by lionleaf** [comment]
Thanks, @stianjensen !

@ayunami2000 How does it look now? Did that solve the issues you were seeing?

**Comment by lionleaf** [comment]
Reopening to make it visible until people have tried it on various phones

**Comment by ayunami2000** [comment]
The navigation menu can be click on even when it is closed still. Other than that, it looks like it's been fixed!

**Comment by ayunami2000** [comment]
Just found another bug... Sometimes the comments as well as the code will just be one big blank space. Also, it seems as if some freeze up, but the page runs fine. Maybe Dwitter uses async loading functions?

**Comment by ayunami2000** [comment]
Here are some screenshots:
Me scrolling down:
1) ![1](https://s18.postimg.org/72c2uhdq1/32_EC1_AE8-_C893-408_E-_B672-_D9_B1_CF67_E774.jpg)
2) ![2](https://s18.postimg.org/fkliytryh/16_AC6_FAC-77_D9-4045-9_BA9-4_E79_C7_FB0_D5_B.jpg)
3) ![3](https://s18.postimg.org/84m9d1byx/928_E2046-_CA25-468_D-_AA0_D-_AE2_B2_C2_A475_F.jpg)
This one actually wasn't moving on the page like it should have been, so this still photo is accurate:
![lag](https://s18.postimg.org/ofmd9cw6h/55_CFA00_F-1_E37-4264-8_DF6-_ABC90_E53_F684.jpg)

**Comment by ayunami2000** [comment]
I think some of this is b/c of the dark theme:
![img](https://s18.postimg.org/kknkkvpnd/D028_D213-_B7_C8-4899-_AB17-02934_B6_F787_C.jpg)

**Comment by dwitterer** [comment]
Well if it is to do with the dark theme, then this PR would fix that too, since the dark theme would be basically an extension of the main theme.

**Comment by ayunami2000** [comment]
Yeah but what I mean is that the dark theme didn't change the font color in a few places

**Comment by stianjensen** [comment]
The annoying issue with the big white square suddenly taking over the screen is still there. I'll see if I kind find some time to investigate that over the weekend!

**Comment by dwitterer** [comment]
How very strange, Dark Theme works just fine on desktop, in all places... So must be an issue with iPhones/mobile...

**Comment by lionleaf** [comment]
#156

**Comment by stianjensen** [comment]
> The navigation menu can be click on even when it is closed still. Other than that, it looks like it's been fixed!

PR for this now: #305

**Comment by stianjensen** [comment]
Not sure if the rest of the issues noted here are related to #156 or not, so I'm gonna try and reproduce some more after #306 has landed.

**Comment by stianjensen** [comment]
I think of all of the issues mentioned in this thread are fixed now. Both the big white square and clicking the navigation menu while visible.

**Comment by ayunami2000** [comment]
The only things left are the white preview on the dark theme background. The actual posts have a white background, while the rest of the page has a dark one. Also the navigation area doesn't open now.

**Comment by stianjensen** [comment]
Oh yeah, that's right the navigation area doesn't open on mobile. I'll have a look at that.
